nexus/lang/build.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```

Comment:
there were a couple issues with this script for me, fixed here: https://github.com/wandb/wandb/pull/6105/commits/c42525413dcc833e4ca8b93f7e4ad213ff7d0b98

my suggestion is to move it all to the newly added noxfile.py in the root of the repo and split it into multiple sessions (nox speak). nox is great, let's use nox more

Reply:
doing that in a next pr is fine

Reply:
yes. used clang-format
thanks for the changes... im wondering why my toolchain didnt require the c++17 options.. they should have been there.
For sure, the build is super preliminary, nox might be a great choice.